### PR TITLE
vili: Initial support

### PR DESCRIPTION
**Pull request by swiitchOFF**

### CODE REVIEW

File: vili.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "Xiaomi 11T Pro",
+			"filename": "crDroidAndroid-15.0-20241109-vili-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/vili/11.x/crDroidAndroid-15.0-20241109-vili-v11.0-BETA.zip/download",
+			"timestamp": 1731183210,
+			"md5": "783414c431391bd87f8800aa76cd9236",
+			"sha256": "c2c2db68eef354131afa11ace8678a39f21c9458ab2499a669fe879731e16640",
+			"size": 1950477179,
+			"version": "11.0",
+			"buildtype": "BETA",
+			"forum": "https://xdaforums.com/t/rom-oss-15-0-official-crdroid-v11-x-for-xiaomi-11t-pro-vili.4702137/",
```
Comment (gwolf2u) [inline vili.json:15]: fix url for crDroid Device Group and Support Community

File: vili.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "https://nikgapps.com",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/vili/11.x/recovery/",
+			"paypal": "",
+			"telegram": "https://t.me/swiitchOFF_updates",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_vili",
+			"common-dt": "https://github.com/crdroidandroid/android_device_xiaomi_sm8350-common",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_sm8350"
```
Comment (gwolf2u) [inline vili.json:25]: push kernel branch

File: vili_changelog.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Build type: Signed
```
Comment (gwolf2u) [inline vili_changelog.txt]: BETA

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: vili.json (hunk 1 above)
Comment (swiitchOFF) [reply vili.json:15]: Fixed

File: vili.json (hunk 2 above)
Comment (swiitchOFF) [reply vili.json:25]: Pushed

File: vili_changelog.txt (hunk 3 above)
Comment (swiitchOFF) [reply vili_changelog.txt]: Fixed

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**